### ISSUE TITLE
is_nodata(NaN)  when NaN is used as no-data value  ?

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-processes#361]
In our geotrellis implementation IEEE754 NaN is used as no-data value for float raster cubes.

This creates a conflict in the definition of `is_nodata(NaN)`: https://github.com/Open-EO/openeo-processes/blob/2cfd022fdc4826af89166d88880a873cbea20e90/is_nodata.json#L4

> data is missing data, i.e. equals to null or any of the no-data values

=>  `is_nodata(NaN) = true`

> `The special numerical value NaN as defined by IEEE Standard 754 is not considered no-data and must return false.

=>  `is_nodata(NaN) = false`

**Comment by m-mohr** [comment]
Yeah, so this was meant to say that NaN is not implicitly also a no-data value as some may have understood it. Of course, it should possible that NaN is a nodata value if explicitly specified as such. I'll issue a PR to fix this.
